hydrus/helpers.py

```diff
@@ -530,3 +530,21 @@ def parse_collection_members(object_: dict) -> dict:
             return error_response(error)
     object_['members'] = members
     return object_
+
+
+def get_fragments(resource: str) -> dict:
+    """Gets a fragment of the main hydra vocabulary.
+
+    :param resource: Fragment specified in the request parameters
+    :type resource: str
+    :return: Object referred to by the fragment
+    :rtype: dict
+    """
+    resourcedict = dict()
```

Comment:
I think it might be better to rename the parameter `resourcedict` to `resource_dict` for better readability and keeping in line with PEP8 standard

Reply:
Sure! I will make that change